components/apimgt/org.wso2.carbon.apimgt.common.analytics/src/main/java/org/wso2/carbon/apimgt/common/analytics/collectors/impl/SuccessRequestDataCollector.java

```diff
@@ -39,6 +40,7 @@ public class SuccessRequestDataCollector extends CommonRequestDataCollector impl
     private static final Log log = LogFactory.getLog(SuccessRequestDataCollector.class);
     private RequestDataPublisher processor;
     private AnalyticsDataProvider provider;
+    private Boolean isResponseSizeRequested = false;
```

Comment:
I think we can remove this variable `isResponseSizeRequested` if it is not used anywhere in the code.